ddtrace/appsec/_ddwaf/waf.py

```diff
@@ -51,11 +51,11 @@ def __init__(
         ruleset_map: Dict[str, Any],
         obfuscation_parameter_key_regexp: bytes,
         obfuscation_parameter_value_regexp: bytes,
+        metrics,
```

Comment:
can we use an interface for typing?

Reply:
can't we create an [interface file](https://github.com/DataDog/dd-trace-py/compare/christophe-papazian/remove_static_cycles_in_appsec...ramy.elkest/remove_static_cycles_in_appsec?expand=1)?

we may have to move it for the circular deps